### PR TITLE
feature: add role to invites

### PR DESCRIPTION
**Pull request by skeptrunedev**

### CODE REVIEW

File: convenience.sh [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
 reset_qdrant_database() {
     echo "Resetting the Qdrant database..."
-    docker compose stop qdrant-database
-    docker compose rm -f qdrant-database
-    docker volume rm arguflow_qdrant_data
-    docker compose up -d qdrant-database
+    sudo docker compose stop qdrant-database
+    sudo docker compose rm -f qdrant-database
+    sudo docker volume rm arguflow_qdrant_data
+    sudo docker compose up -d qdrant-database
```
Comment (skeptrunedev) [inline convenience.sh:9]: @fastfists what's the command to fix this again?

Comment (fastfists) [reply convenience.sh:9]: sudo usermod -aG docker <user-name>

File: server/src/handlers/auth_handler.rs [hunk 2]
```diff
@@ -209,48 +209,14 @@ pub async fn create_account(
...
-                "This organization is not registerable".to_string(),
-            ));
-        }
+    if let Some(inv_code) = inv_code {
+        let invitation =
+            check_inv_valid(inv_code, email.clone(), organization_id, pool.clone()).await?;
+        role = invitation.role.into();
     }
 
     let user_org =
```
Comment (fastfists) [inline server/src/handlers/auth_handler.rs:218]: Should we be creating a user each time?

**Review by fastfists — APPROVED** [review]

File: convenience.sh (hunk 1 above)
Comment (fastfists) [inline convenience.sh:9]:
```suggestion
    docker compose stop qdrant-database
    docker compose rm -f qdrant-database
    docker volume rm arguflow_qdrant_data
    docker compose up -d qdrant-database
```

**Review by fastfists — APPROVED** [review]

**Pull request merged**